project-base/storefront/helpers/errors/mapGraphqlErrorForDevelopment.ts

```diff
@@ -0,0 +1,11 @@
+import { GraphQLError } from 'graphql';
+
+export const mapGraphqlErrorForDevelopment = (graphqlError: GraphQLError) => ({
+    message: graphqlError.message,
+    location: graphqlError.locations,
+    path: graphqlError.path,
+    extensions: {
+        ...graphqlError.extensions,
+        trace: Array.isArray(graphqlError.extensions.trace) ? graphqlError.extensions.trace.slice(0, 3) : [],
```

Comment:
I wonder what are other possible types of `graphqlError.extensions.trace`? Cannot this be `graphqlError.extensions.trace?.slice(0, 3) || []`?

Reply:
Its unknown 🙃 So I think this is the best way to check it